cylc/flow/network/schema.py

```diff
@@ -867,330 +882,622 @@ class Meta:
     res = await resolvers.nodes_mutator(info, command, ids, w_args, args)
     return GenericResponse(result=res)
 
+# Input types:
+
+
+class WorkflowName(String):
+    """a"""
+
+
+class User(String):
+    """a"""
+
+
+class WorkflowID(InputObjectType):
+    """A workflow identifier of the form `user|workflow_name`."""
+    name = WorkflowName(required=True)
+    user = User()
+
+
+class CyclePoint(String):
+    """An integer or date-time cyclepoint."""
+
+
+class CyclePointGlob(String):
+    """An integer or date-time cyclepoint."""
+
+
+class BroadcastSetting(InputObjectType):
+    """A s"""
+
+    key = String(
+        description=sstrip('''
+            The cylc namespace for the setting to modify.
+            e.g. `[task_name][environemt]variable_name`.
+        '''),
+        required=True
+    )
+    value = String(
+        description='The value of the modification'
+    )
+
+
+TaskStatus = Enum(
+    'TaskStatus',
+    [
+        (status.replace('-', '_'), status)
+        for status in TASK_STATUSES_ALL
+    ]
+    # description='The status of ac task e.g. waiting, running, succeeded.'
+)
+
+
+class TaskState(InputObjectType):
+    """The state of a task, a combination of status and other field."""
+
+    status = TaskStatus()
+    is_held = Boolean(description=sstrip('''
+        If a task is held no new job submissions will be made
+    '''))
+
+
+class TaskName(String):
+    """The name a task.
+
+    * Must be a task not a family.
+    * Does not include the cycle point.
+    * Any parameters must be expanded (e.g. can't be `foo<bar>`).
+    """
+
+
+class NamespaceName(String):
+    """The name of a task or family."""
 
-# Mutations defined:
-class ClearBroadcast(Mutation):
+
+class NamespaceIDGlob(InputObjectType):
+    """A glob search for an active task or family."""
+
+    cycle = CyclePointGlob()
+    namespace = NamespaceName()
+    status = TaskState()
+
+
+class TaskID(InputObjectType):
+    """The name of an active task."""
+
+    cycle = CyclePoint(required=True)
+    name = TaskName(required=True)
+
+
+class JobID(InputObjectType):
+    """A job submission from an active task."""
+
+    task = TaskID(required=None)
+    submission_number = Int(default=-1)
+
+
+class TimePoint(String):
+    """A date-time in the ISO8601 format."""
+
+# Mutations:
+
+
+class Broadcast(Mutation):
     class Meta:
-        description = """Expire all settings targeting cycle points
-earlier than cutoff."""
-        resolver = mutator
+        description = sstrip('''
+            Override [runtime] config in targeted namespaces in a running
```

Comment:
It doesn't just "Override", it can add new.

Reply:
Good point.

BTW, what I've done here is to copy in the CLI doc. Once this is all done the CLI will get generated from this spec so we only need to update the doc in the schema.